### DISCUSSION TITLE
https://ais-web.test.dp.tech/datasets/detail?name=C12H26-dodecane&id=11&pageType=datasets

### DISCUSSION THREAD

**Discussion by giscus** [yanyumei/giscus-test#12] [Announcements]
# https://ais-web.test.dp.tech/datasets/detail?name=C12H26-dodecane&id=11&pageType=datasets

https://ais-web.test.dp.tech/datasets/detail?name=C12H26-dodecane&id=11&pageType=datasets

**Comment by smallplum** [comment]
测试一下

**Comment by smallplum** [comment]
哦哦哦
